app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -13,8 +13,14 @@ def initialize(enrollment:, barcode_image_url: nil)
         @barcode_image_url = barcode_image_url
       end
 
+      # Reminder of days left is exclusive of the day the email is sent, hence the reminder num is 1 less than days_to_due_date
+      def days_reminder
```

Comment:
I feel like this might be more clearly named as `days_remaining`? since the result is a number?
```suggestion
      def days_remaining
```